specifyweb/frontend/js_src/lib/components/Forms/ResourceView.tsx

```diff
@@ -314,7 +316,7 @@ export function ResourceView<SCHEMA extends AnySchema>({
         )
       }
       className={{
-        container: `${dialogClassNames.normalContainer} ${
+        container: `${dialogClassNames.normalContainer} ${classNameProp} ${
```

Comment:
`classNameProp` could be undefined - and then it will literally add a `"undefined"` class name - eslnit can prevent this bug

Reply:
in the video you sent of Accession dialog, is the dialog manually resized?

With `w-[inherit]`, the dialog is full width for me always (because the parent div that the dialog would inherit the width from has full width) - the effect is the same as if I had `w-full` - thus, can you try out `w-full` instead of `w-[inherit]`

moreover, consider the variable `isFullHeight` in this file - consider renaming it to `isFullSize` and when true, adding both `h-full` and `w-full` to the dialog class names - that would remove the need for a separate classNameProp, and will fix this bug in more places

Reply:
'in the video you sent of Accession dialog, is the dialog manually resized?' 
No it's not